samples/amplitude-sample/src/app/app.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="container" style="text-align:center; max-width:850px">
+  <h1>
+    <b>Welcome to ConfigCat + Amplitude A/B Testing Sample app!</b>
+  </h1>
+  <h4>
+    
+  </h4>
+  <p>
+    We would like to show a method of how to A/B testing with ConfigCat and Amplitude. This sample belongs to a <a href="https://configcat.com/blog" target="_blank">blog post</a>
```

Comment:
This is a sample app to demonstrate how to do A/B testing with Amplitude and ConfigCat. Here is a detailed article about ...